tasks/serializers.py

```diff
@@ -10,7 +10,8 @@ class CreateTaskSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Task
-        fields=['social_media_platform', 'share_type', 'share_link', 'share_text', 'share_img']
+        fields = [
+            'social_media_platform', 'description', 'share_type', 'share_link', 'share_text', 'share_img']
```

Comment:
OMG :rofl: why but?

Reply:
This is what happens when linting messes with you I guess 😂😂

Reply:
but description wasn't needed right?? :laughing:

Reply:
Lets go back to where I used the rare and meaningful word "necessity" 😆